_events/2016-05-summit-boulder.md

```diff
@@ -11,6 +11,8 @@ poster_hero: "/img/media/boulder.jpg"
 
 location_section: true
 papers_section: true
+
+featured: true
```

Comment:
hah, no, i was stabbing randomly at that one, don't know what it means. I'll take it out